### PR TITLE
Bump tee-tools crates

### PR DESCRIPTION
**Pull request by grydz**

### CODE REVIEW

**Review by ThibsG — APPROVED** [review]

**Review by Manuthor — APPROVED** [review]

File: crate/agent/src/cloud_detection.rs [hunk 1]
```diff
@@ -0,0 +1,67 @@
+use std::time::Duration;
+
+use cosmian_vm_client::cloud_provider::CloudProvider;
+
+const METADATA_URL: &str = "http://169.254.169.254";
```
Comment (Manuthor) [inline crate/agent/src/cloud_detection.rs:5]: LGTM!
Just one question, how do you configure the `METADATA_URL`?

File: crate/cli/src/verify.rs [hunk 2]
```diff
@@ -87,10 +88,16 @@ impl VerifyArgs {
         };
 
         let mut policy = snapshot.tee_policy;
-        policy.set_report_data(&forge_report_data_with_nonce(
-            &nonce,
-            &client.certificate.0,
-        )?)?;
+
+        if let Some(cloud_type) = snapshot.cloud_type {
+            if cloud_type != CloudProvider::Azure {
```
Comment (ThibsG) [inline crate/cli/src/verify.rs:93]: can you just add a small comment explaining why we avoid `set_report_data` with Azure?

**Review by ThibsG — APPROVED** [review]

File: crate/agent/src/cloud_detection.rs (hunk 1 above)
Comment (grydz) [reply crate/agent/src/cloud_detection.rs:5]: You don't. This is the same server's IP on all cloud provider!

File: crate/cli/src/verify.rs (hunk 2 above)
Comment (grydz) [reply crate/cli/src/verify.rs:101]: Done in e1271e9082932524d100ef95f75439d8157658f8.

**Pull request merged**